apps/web/src/components/ScreeningDecisions/ScreeningDecisionDialogForm.tsx

```diff
@@ -52,9 +54,9 @@ const ScreeningDecisionDialogForm = ({
   const isAssessmentDecisionSuccessful =
     watchAssessmentDecision === AssessmentDecision.Successful;
   const isAssessmentDecisionUnSuccessful =
-    watchAssessmentDecision === AssessmentDecision.Unsuccessful;
-  const isAssessmentDecisionNotSure =
-    watchAssessmentDecision === AssessmentDecision.NotSure;
+    watchAssessmentDecision === AssessmentDecision.Unsuccessful ||
```

Comment:
I don't think you can double these up.  For WATCH you need to reset justifications, I think.
![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/8978655/08e150e2-eb17-4443-a93a-d2b561599777)

Reply:
Okay, I was a little confused by this specific part and I was looking into this and think I messed it up. It seems like maybe this auto sets it to "other" and then shows the notes? 🤔 

![Screenshot 2024-01-08 153009](https://github.com/GCTC-NTGC/gc-digital-talent/assets/4127998/60fbb0cd-4a98-4d52-ba13-6fb2fe92749b)

Reply:
Oh yes, you're right!  There should be the copy block and "Other reason" input.

Reply:
Alright, I _think_ I got it in [update justification logic for on hold decision](https://github.com/GCTC-NTGC/gc-digital-talent/pull/8955/commits/27e492e2c0e33e11cc1abbfada27ae3012023a4e)

Reply:
Hmm, that's an interesting way to do it - set the other justification in a hidden list.  I don't mind, I guess it makes sense.